test/data/manifests/rhel_84-x86_64-qcow2-boot.json

```diff
@@ -3517,6 +3517,7 @@
             "type": "grub2"
           },
           "format": "qcow2",
+          "qcow2_compat": "0.10",
```

Comment:
Hmm, are you sure you aren't comparing values between rhel8 and rhel84? I think 0.10 is used everywhere for rhel84.

Reply:
I am comparing rhel8 vs rhel84 but why are they different ?

Reply:
`rhel8` is in fact RHEL 8.3 and we don't change distro definitions for RHEL's minor releases that are already released. We're only changing RHEL 8.4 because we have a `blocker+`.

Reply:
alright, got it.